apps/projects/templates/meinberlin_projects/includes/phase_indicator.html

```diff
@@ -0,0 +1,43 @@
+{% load contrib_tags i18n %}
+
+{% with project.active_phase as active_phase %}
+
+<div class="phaseList">
+    <ul>
+        {% with project.phases|length as phase_count %}
+        {% for phase in project.phases %}
+            {% includeTemplateString "meinberlin_projects/includes/phase_popover.html" phase=phase phase_num=forloop.counter phase_count=phase_count as popover_content %}
+            <li>
+                <button
+                    title="{% trans phase.name %}"
```

Comment:
Is this marked as translatable somewhere else? Otherwise this will not be extracted.